dev-resources/template/1_db.yml

```diff
@@ -58,11 +58,20 @@ Parameters:
     AllowedValues:
       - true
       - false
-    Default: true
+    Default: false
```

Comment:
The default for `DBDeletionProtection` should absolutely be `true`

Reply:
addressed via https://github.com/yetanalytics/lrsql/pull/276/commits/f4112ab6ffa5608f7f85a6ef1460f5c4ab9aefd3